core/tests/protractor_utils/TopicsAndSkillsDashboardPage.js

```diff
@@ -140,14 +140,13 @@ var TopicsAndSkillsDashboardPage = function() {
     });
   };
 
-  this.createTopic = function(title, abbreviatedName) {
+  this.createTopic = function(title) {
     waitFor.elementToBeClickable(
       createTopicButton,
       'Create Topic button takes too long to be clickable');
     createTopicButton.click();
 
     topicNameField.sendKeys(title);
-    abbreviatedTopicNameField.sendKeys(abbreviatedName);
```

Comment:
If we aren't going to use the abbreviated name anymore, let's also remove the `abbreviatedTopicNameField` variable from the top of the file

Reply:
The plan for this PR was to just remove it from the UI so that topic creation can continue in the test and prod server. The complete removal of the field is now being tracked by #9102

Reply:
@aks681 Why though are we leaving the `abbreviatedTopicNameField` variable at the top of this file? I think we're removing its only usage, right?

Reply:
Ah, yeah, done.